content/mqtt/index.textile

```diff
@@ -0,0 +1,103 @@
+
+h3(#token-auth). Token authentication
+
+For token authentication you'll be required to "create a token from your own servers":https://www.ably.io/documentation/general/authentication#token-authentication. Once you have the token, you can simply pass it through when trying to connect to Ably as the connection's @username@ and leaving the @password@ empty. In the future we will be adding "JWT":https://jwt.io/ (JSON Web Tokens) as a form of authentication.
+
+If using token authentication, you can subscribe to a special topic, *[mqtt]tokenevents*, to get a warning when the current connection's token is about to expire. This will be a single message, sent 30 seconds before the token expires, with the 13 byte payload *expirywarning*.
+
+On receiving this, the client is recommended to get a new token, then disconnect and reconnect with the new token themselves. If this is not done, once the token expires, the server will abruptly disconnect the connection.
+
+h3(#ssl). SSL
+
+We support both ssl and non-ssl connections (which use a different port, see above), but strongly recommend using ssl wherever possible. If you are not using ssl, note that the same restrictions apply as if you were using an Ably client without ssl. That is, connection attempts using basic authentication (i.e. an api key) are disallowed, and any namespaces which you've enabled 'require TLS' on will be inaccessible. If this is a problem for you, please do let us know at support@ably.io and we'll see what we can do.
+
+h2(#info). Extra information
+
+h3(#adapter-info). Adapter information
+
+* A protocol adapter inevitably adds some latency. Using the adapter will be a little slower than using Ably native client libraries. Typically the impact is in the low milliseconds
+* Behind the scenes, the adapter just uses the normal Ably service, so there is no problem with using MQTT and Ably client libraries side by side. You can mix and match as you like; for example, using MQTT on your IoT devices, but using the Ably Realtime API on your servers
+* While the adapter can be useful for devices which need to use MQTT, there are many benefits to using the Ably client libraries (e.g. "continuity guarantees":https://support.ably.io/solution/articles/3000044639-connection-state-recovery, "fallback host support":https://support.ably.io/solution/articles/3000044636-routing-around-network-and-dns-issues, "history":https://www.ably.io/documentation/realtime/history, "presence":https://www.ably.io/documentation/realtime/presence) require the use of the Ably client libraries. As a result, if a native Ably library is available for your platform, we recommend you consider using the Ably client libraries instead or at least make a plan to eventually transition over to Ably native client libraries
```

Comment:
- `there are many benefits to using the Ably client libraries (e.g. ...) require the use of the Ably client libraries` doesn't scan

- "at least make a plan to eventually transition over to Ably native client libraries" -- I'm guessing you copied this over from a Pusher or PUbnub usage adapter article, it doesn't really make sense for mqtt as that's not targeted at helping people switch from a competitor